dask/dataframe/io/orc.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import, division, print_function
+
+from ..core import DataFrame
+from ...base import tokenize
+from ...bytes.core import get_fs_token_paths
+from ...utils import import_required
+from .parquet import _get_pyarrow_dtypes, _meta_from_dtypes
+from ...compatibility import string_types
+
+__all__ = ('read_orc',)
+
+
+def _read_orc_stripe(fs, path, stripe, columns=None):
+    """Pull out specific data from specific part of ORC file"""
+    orc = import_required('pyarrow.orc', 'Please install dev version pyarrow')
```

Comment:
Obviously should replace with a version check, when a release for orc is decided.

Reply:
Is there now an explicit version to point to?